_includes/user-menu.html

```diff
@@ -42,6 +42,7 @@
 		<ul>
             <li><a href="{{docu}}/features/index.html">Overview</a></li>
             <li><a href="{{docu}}/features/sitemap.html">Sitemaps</a></li>
+            <li><a href="{{docu}}//features/things-items.html">Things and Items</a></li>
             <li><a href="{{docu}}/features/persistence.html">Persistence</a></li>
```

Comment:
There is one slash too many, "Things and Items" should be before Sitemaps
Reference: https://community.openhab.org/t/documenting-openhab-2/10568/28
Maybe you can also check this article to see, what's still missing.